src/charm.py

```diff
@@ -250,46 +252,28 @@ def _get_n4_upf_hostname(self) -> str:
             return lb_hostname
         return self._upf_hostname
 
-    def _network_attachment_definitions_from_config(self) -> list[NetworkAttachmentDefinition]:
-        """Returns list of Multus NetworkAttachmentDefinitions to be created based on config."""
-        access_nad_config = {
-            "cniVersion": "0.3.1",
-            "ipam": {
-                "type": "static",
-                "routes": [
-                    {
-                        "dst": self._get_gnb_subnet_config(),
-                        "gw": self._get_access_network_gateway_ip_config(),
-                    },
-                ],
-                "addresses": [
-                    {
-                        "address": self._get_access_network_ip_config(),
-                    }
-                ],
-            },
-            "capabilities": {"mac": True},
-        }
-        if (access_interface := self._get_access_interface_config()) is not None:
+    def _network_attachment_definitions_from_config(
+        self,
+    ) -> Optional[list[NetworkAttachmentDefinition]]:
```

Comment:
```suggestion
    ) -> list[NetworkAttachmentDefinition]:
```
The `Optional` type hint was left behind